app/views/audits/show.html.erb

```diff
@@ -23,42 +23,44 @@
     <% end %>
     <div class='tab-pane <%= "active" unless tmplt%>' id="tab1">
       <table class="<%= table_css_classes %>">
-        <thead>
-          <tr>
-            <th><%= _('Item') %></th>
-            <% if @audit.action == 'update' %>
-              <th><%= _('Old') %></th>
-              <th><%= _('New') %></th>
-            <% else %>
-              <th><%= _('Value') %></th>
-            <% end  %>
-          </tr>
-        </thead>
+        <%if @audit.audited_changes.key?("template") && @audit.audited_changes.length==1 %>
```

Comment:
please add a space after `<%` and around `==`, also I think `@audit.audited_changes.size` is prefered